charts/victoria-metrics-anomaly/templates/configmap.yaml

```diff
@@ -4,5 +4,8 @@ metadata:
   name: {{ include "chart.fullname" . }}
   namespace: {{ include "vm.namespace" . }}
   labels: {{- include "chart.labels" . | nindent 4 }}
+  {{- with .Values.config.annotations }}
```

Comment:
please change this parameter to `.Values.configMeta.annotations` to avoid having these parameters in configuration file

Reply:
Can you elaborate more What is `configMeta` ? What do you think about  annotations in the root level to be `configMapAnnotations` similar to `podAnnotations` ?

Reply:
```
What do you think about annotations in the root level to be configMapAnnotations similar to podAnnotations
```
this also works

Reply:
I changed it , Could you take a look